README.md

```diff
@@ -37,6 +37,8 @@ use opentelemetry::{exporter::trace::stdout, trace::Tracer};
 
 fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
     // Create a new instrumentation pipeline
+    // Note: using `_` instead of assigning to a variable will immediately
+    // drop the tracer and uninstall it so make sure to properly assign it.
```

Comment:
May be clearer to have this say that it will drop the uninstall guard and make the tracer a no-op as the tracer won't be dropped unless you assign it to `_` as well.

Reply:
Ah yes, that is maybe not crystal clear.